firstvoices/backend/tests/test_apis/test_my_sites_api.py

```diff
@@ -125,3 +125,22 @@ def test_language_admin_role(self):
         assert response.status_code == 200
         response_data = json.loads(response.content)
         assert response_data["count"] == 3
+
+    @pytest.mark.django_db
+    @pytest.mark.parametrize(
+        "role", [Role.MEMBER, Role.ASSISTANT, Role.EDITOR, Role.LANGUAGE_ADMIN]
+    )
+    def test_hidden_sites_visible(self, role):
+        user = factories.UserFactory.create()
+        hidden_site = factories.SiteFactory.create(
+            visibility=Visibility.PUBLIC, is_hidden=True
+        )
+        factories.MembershipFactory.create(user=user, site=hidden_site, role=role)
```

Comment:
This API needs to use the permission you created-- users with the MEMBER role should not be able to see hidden sites via this API